test/integration/agent-integration.test.ts

```diff
@@ -78,7 +78,7 @@ describe('Integration test', () => {
       expect(domSnapshot).contains('badssl.com')
     })
 
-    it('snapshots a complex website with responsive images', async () => {
+    it.skip('snapshots a complex website with responsive images', async () => {
```

Comment:
I think we can just delete this test tbh. We have a responsive image test further down that isn't a live site.

Reply:
💯